src/app/Scenes/Activity/components/NotificationArtworkList.tsx

```diff
@@ -29,6 +31,9 @@ export const NotificationArtworkList: FC<NotificationArtworkListProps> = (props)
   const artworksConnectionData = useFragment(notificationArtworkListFragment, artworksConnection)
 
   const artworks = extractNodes(artworksConnectionData)
+  const note = partnerOffer?.note
```

Comment:
suggestion(nitpick - feel free to ignore): 
```suggestion
  const { note } = partnerOffer
```

Reply:
I needed to add a null check still 🤔